include/maidsafe/crux/detail/sequence_number.hpp

```diff
@@ -77,8 +78,8 @@ sequence_number<NumericType, Max>::sequence_number(const sequence_number& other)
 }
 
 template<typename NumericType, NumericType Max>
-sequence_number<NumericType, Max>::sequence_number(NumericType n)
-    : n(n)
+sequence_number<NumericType, Max>::sequence_number(NumericType other_n)
+    : n(other_n)
```

Comment:
Am I correct to assume that this has nothing to do with the warning? The original code is both valid and correct.